include/effects/matrix/PatternLife.h

```diff
@@ -117,7 +117,7 @@ class PatternLife : public LEDStripEffect
     // Seed: 92465, Generations: 1626
 
     static constexpr long bakedInSeeds[] =

```

Comment:
Are these new seeds you discovered or where's this set of a few come from?

Reply:
I did not touch this code. I think they're coming from a set of commits I pulled in mid-way from your fork?